WeRadiate-ThermoSense.ino

```diff
@@ -405,6 +464,214 @@ void loop(void)
         {
         // put your main code here, to run repeatedly:
         gCatena.poll();
-        Serial.println("hello world");
-        delay(60000);
+        }
+
+void startSendingUplink(void)
+        {
+        TxBuffer_t b;
+        LedPattern savedLed = gLed.Set(LedPattern::Measuring);
+
+        b.begin();
+        FlagsSensor3 flag;
+
+        flag = FlagsSensor3(0);
+
+        b.put(FormatSensor2); /* the flag for this record format */
```

Comment:
You can't use `FormatSensor2` with `FlagsSensor3()`.  You need to use `FormatSensor3`.  If any of the formats don't fit your needs from an engineering units point of view, we need a new format. But we have plenty of formats with the OneWire temperature so you should be able to use an existing format..